apps/frontend/src/components/BlogNav.tsx

```
@@ -58,45 +59,84 @@ export default function BlogNav() {
+
+                    <input
+                        type="checkbox"
+                        onChange={handleChange}
+                        id="search-toggle"
+                        ref={search}
+                        aria-expanded={search?.current?.checked || false}
+                        className="peer/search hidden"
+                    />
+                    <label
+                        htmlFor="search-toggle"
+                        role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)